gtest/CMakeLists.txt.in

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.2)
+
+project(gtest NONE)
+
+include(ExternalProject)
+ExternalProject_Add(gtest
+  GIT_REPOSITORY    https://github.com/google/googletest.git
+  GIT_TAG           master
```

Comment:
Maybe we'd want to allow to specify a version for gtest?

I know they plan to follow the "live at head" philosophy but that might be a burden on the maintainer of jrl-cmakemodules-enabled project that simply wants its tests to keep working.

Reply:
I just added an optional argument to ADD_GTEST_SUITE to account for a precise tag.